src/tests/ftest/control/dmg_pool_query_ranks.yaml

```diff
@@ -1,20 +1,26 @@
 hosts:
-  test_servers: 3
+  test_servers: 5
 timeouts:
   test_pool_query_ranks_basic: 120
   test_pool_query_ranks_error: 120
   test_pool_query_ranks_mgmt: 480
 server_config:
   name: daos_server
+  crt_timeout: 5
   engines_per_host: 1
   engines:
     0:
       targets: 4
       nr_xs_helpers: 0
+      env_vars:
+        - SWIM_SUSPECT_TIMEOUT=10000
+        - DAOS_POOL_RF=1
+        - DD_MASK=io,epc,rebuild
       storage:
         0:
           class: ram
           scm_mount: /mnt/daos
   system_ram_reserved: 1
 pool:
   size: 4GB
+  svcn: 5
```

Comment:
I think better to not set svcn, since the default will be 5 with 5 servers? This has been an issue before, so we've started removing svcn from pool create and letting DAOS handle it

Reply:
This is written on purpose, i want to simplify codes, otherwise i need take care in the codes don't break 1/2 of PS replica, other pool query won't work.